rcaracaus/pledge-vegan-node
# Made the topic listing page functional (4)[6]

Steps to test

1. Go to a node page http://localhost:3000/silverback-gorilla-shatters-glass-little-girl-beats-her-chest
2. Click on the field tag which will link you to a topic listing http://localhost:3000/topic/animals
3. Check if you get a page listing with a title at the top with the name of the topic, 5 listings with an image, a title and a body with stripped html.
4. Go to http://localhost:3000/topic/animals directly to see if it works without coming from a node detail page.

--

To do:

- [x] Don't forget to rename to topic-detail
- [x] Strip tags
